app/src/main/java/com/nitrr/ecell/esummit/ecellapp/misc/MySnapHelper.java

```diff
@@ -0,0 +1,53 @@
+package com.nitrr.ecell.esummit.ecellapp.misc;
+
```

Comment:
Why there is need  to make this class? 
It's inbuilt in the Android package. (SnapHelper)
Use direct implementation, or state the reason if necessary.

Reply:
SnapHelper is an abstract class. For direct implementations, there are inBuilt classes LinearSnapHelper and PagerSnapHelper. In LinearSnapHelper, when we swipe, it scrolls more than one views. I have edited it such that when swiped once, it will move to the adjacent view only.